src/key/dynamic.rs

```diff
@@ -1,25 +1,22 @@
 #[cfg(not(feature = "std"))]
-use alloc::Clone;
-#[cfg(not(feature = "std"))]
-use alloc::String;
-#[cfg(not(feature = "std"))]
 use alloc::borrow::Cow;
 #[cfg(not(feature = "std"))]
+use alloc::string::String;
+#[cfg(not(feature = "std"))]
 use alloc::string::ToString;
 #[cfg(not(feature = "std"))]
+use core::clone::Clone;
+#[cfg(not(feature = "std"))]
 use core::cmp::PartialEq;
 #[cfg(not(feature = "std"))]
-use core::convert::{AsRef, From, Into};
+use core::convert::{AsRef, From};
 #[cfg(not(feature = "std"))]
 use core::fmt;
 #[cfg(not(feature = "std"))]
 use core::hash::{Hash, Hasher};
 #[cfg(not(feature = "std"))]
 use core::iter::{FromIterator, IntoIterator};
-#[cfg(not(feature = "std"))]
-use core::ops::Deref;
```

Comment:
Hmm. It appears we don't implement the Deref change in this file.

Thanks for noticing the unused imports!